### DISCUSSION TITLE
can write by one  programming language and read by others

### DISCUSSION THREAD

**Discussion by xtuyaowu** [erthink/libmdbx#334] [Q&A] [answered]
hi guys libmdbx can write by one  programming language and read by others?
such as write by java and read by python、c++、rust ？if can , where is python、rust example.

thanks!

**Comment by erthink** [comment, marked as answer]
Yes, of course.
Obliviously you should use the same binary data format, i.e. the same (de)serialization method(s) for your keys and values, etc.

**Comment by xtuyaowu** [comment]
is there any example ？how to use python and rust？

**Comment by erthink** [comment]
> is there any example ？how to use python and rust？

[PRs](https://www.google.com/search?q=What+is+PR+in+GitHub) are welcome!
